modules/fhir-client/src/blaze/fhir_client.clj

```diff
@@ -19,6 +21,29 @@
     :base-uri base-uri}))
 
 
+(defn- password-authenticator [username password]
+  (proxy [Authenticator] []
+    (getPasswordAuthentication []
+      (PasswordAuthentication. username (.toCharArray ^String password)))))
+
+
+(defn- http-client-with-authenticator
+  "Returns a HttpClient with given Authenticator set for authentication."
+  [authenticator]
+  (-> (HttpClient/newBuilder)
+    (.authenticator authenticator)
```

Comment:
Please align arguments of threading macros. Read more about it here: https://github.com/bbatsov/clojure-style-guide#threading-macros-alignment

Reply:
Fixed